src/framework/components/element/system.js

```diff
@@ -392,8 +392,6 @@ class ElementComponentSystem extends ComponentSystem {
         material.emissiveMap = this._defaultTexture;
         material.opacityMap = this._defaultTexture;
         material.opacityMapChannel = 'a';
-        material.opacityTint = true;
-        material.opacity = 0; // use non-1 opacity to compile shader correctly
```

Comment:
did you mean to remove this line? default opacity is 1.

Reply:
yeah it was added to force shader compilation .. later this is set to what it actually needs to be

Reply:
NASTY ALERT, thanks